sklearn/naive_bayes.py

```diff
@@ -523,6 +523,11 @@ def _check_X_y(self, X, y, reset=True):
         return self._validate_data(X, y, accept_sparse="csr", reset=reset)
 
     def _update_class_log_prior(self, class_prior=None):
+        """Update class log priors based `class_prior` (when provided) or class
+        counts.
+
+        This method is called each time `fit` or `partial_fit` update the model.
+        """
```

Comment:
I prefer to follow [numpydoc style guide](https://numpydoc.readthedocs.io/en/latest/format.html#short-summary) even for private methods and keep the short summary at one line:

```suggestion
        """Update class log priors.

        The class log priors are based on `class_prior`, class count or the
        number of classes. This method is called each time `fit` or
        `partial_fit` update the model.
```